src/web-auth/web-message-handler.js

```diff
@@ -58,22 +59,22 @@ WebMessageHandler.prototype.run = function(options, cb) {
   ) {
     var error = err;
```

Comment:
this could be

```
var error = err ? err : eventData.event.data.response.error
if (!error) {
  parsedHash =..
  return validateAuthenticationResponse(...);
}

//continue handling errors
```

Reply:
see `error = eventData.event.data.response;`the `eventData.event.data.response.error` property is a string.